src/scroll-timeline-css-parser.js

```diff
@@ -314,35 +315,62 @@ export class StyleParser {
     const hasViewTimeline = rule.block.contents.includes("view-timeline:");
     const hasViewTimelineName = rule.block.contents.includes("view-timeline-name:");
     const hasViewTimelineAxis = rule.block.contents.includes("view-timeline-axis:");
+    const hasViewTimelineInset = rule.block.contents.includes("view-timeline-inset:");
 
     if(!hasViewTimeline && !hasViewTimelineName) return;
 
-    let viewTimeline = {selector: rule.selector, name: '', axis: 'block'};
+    let timelines = [];
 
     if(hasViewTimeline) {
-      const parts = this.extractMatches(rule.block.contents, RegexMatcher.VIEW_TIMELINE, separator=' ');
-      if(parts.length == 1) {
-        viewTimeline.name = parts[0];
-      } else if(parts.length == 2) {
-        if(TIMELINE_AXIS_TYPES.includes(parts[0]))
-          viewTimeline.axis = parts[0], viewTimeline.name = parts[1];
-        else
-          viewTimeline.axis = parts[1], viewTimeline.name = parts[0];
+      const viewTimelines = this.extractMatches(rule.block.contents, RegexMatcher.VIEW_TIMELINE);
+      for(let tl of viewTimelines) {
+        parts = this.split(tl);
+        let options = {selector: rule.selector, name: '', axis: 'block', inset: null};
+        if(parts.length == 1) {
+          options.name = parts[0];
```

Comment:
Don't we still need to set the axis to block in this case?

Reply:
Confirmed offline that we are setting the default correctly in the view timeline constructor.